### DISCUSSION TITLE
How to save only the date in a DateTime Column

### DISCUSSION THREAD

**Discussion by gliden** [simolus3/drift#3082] [Q&A] [answered]
We have to save only the date in a database. To avoid bugs we don't want to save the time. 
How is this possible?

**Comment by simolus3** [comment, marked as answer]
Do you need the [datetime expression APIs](https://drift.simonbinder.eu/docs/getting-started/expressions/#date-and-time)? If not, I'd suggest defining a [type converter](https://drift.simonbinder.eu/docs/advanced-features/type_converters/) that maps `DateTime` values (or a custom type you have that only supports year-month-day) to strings and define the respective columns as text.

If you do, it gets a little weird but you can define a type converter that maps from `DateTime` to `DateTime` values, apply that to your date time columns and perhaps drop the time part there?
